src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -134,14 +134,23 @@ public MethodSignature MethodSignature
         private FormattableString BuildDescription()
         {
             var pathInformation = _operations.Select(operation =>
-                (FormattableString)$@"<item>
+            {
+                string resourceTypeItem = "";
```

Comment:
please use `string.Empty` when we need an empty string

Reply:
fixed. thanks.